Editor/Window/LandingPage.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using AmazonGameLift.Editor;
+using Editor.Resources.EditorWindow;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace Editor.Window
+{
+    public class LandingPage
+    {
+        private readonly VisualElement _container;
+
+        public LandingPage(VisualElement container)
```

Comment:
We're missing the logic to change the content to display a profile selector when a profile is available

To clarify, I think it would be good for a separate PR to do the profile selector since it's a common component to be pulled into all pages; however, we should still have a placeholder component to test the logic to switch between creation/select functions correctly as part of this PR